src/task1.2/index.js

```diff
@@ -0,0 +1,23 @@
+import { createReadStream, createWriteStream } from 'node:fs';
+import csv from 'csvtojson';
+
+const inputFilePath = './src/task1.2/files/input.csv'
```

Comment:
It's better to use `path.resolve(__dirname,  ... );` to access files. (For example to support different OS).

Reply:
Discussed
```
const dirName = dirname(fileURLToPath(import.meta.url));
const inputFilePath = resolve(dirName, 'files', 'input.csv');
const outputFilePath = resolve(dirName, 'files', 'output.json');
```